modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/components/TextStyles.kt

```diff
@@ -305,6 +305,26 @@ fun TextH70(
     )
 }
 
+@Composable
+fun TextC50(
+    text: String,
+    modifier: Modifier = Modifier,
+    maxLines: Int = Int.MAX_VALUE
+) {
+    Text(
+        text = text.uppercase(Locale.getDefault()),
+        color = MaterialTheme.theme.colors.primaryText02,
+        fontFamily = FontFamily.SansSerif,
+        fontSize = 13.sp,
+        fontWeight = FontWeight.Bold,
```

Comment:
I prefer using `FontWeight.W700` style of constants because I find that to be clearer and easier to match up with the Figma designs. But what do you think? Definitely not a blocker.

Reply:
I picked bold style for `C50` from Figma but don't have any issues in using the other style. 
Replaced in 22ab72a.